### PR TITLE
COMMERCE-10114 Default display page template for a product

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-9433

resent from: #3355 

@alessiorendina

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDisplayPageTemplates.testcase [hunk 1]
```diff
@@ -171,6 +171,143 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-9433 A default display page template for a specific product can override another display page template that is marked as default for all products"
+	@priority = "3"
+	test CanDefaulDPTForSpecificProductOverrideDefaultMarkedDPT {
+		property portal.acceptance = "false";
+
+		task ("Given a Commerce Product type display page template with fragments created and mapped with fields") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDisplayPageTemplates.testcase:179]: Please apply this change to all similar task descriptions
```suggestion
		task ("Given a Commerce Product type display page template with fragments is created and mapped with fields") {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**